datafusion/physical-plan/src/sorts/sort.rs

```diff
@@ -206,8 +201,6 @@ struct ExternalSorter {
     schema: SchemaRef,
     /// Sort expressions
     expr: Arc<[PhysicalSortExpr]>,
-    /// RowConverter corresponding to the sort expressions
-    sort_keys_row_converter: Arc<RowConverter>,
```

Comment:
this was only used for multi-column sorts when one of the columns was `List`. This feature was added to arrow-rs in 
- https://github.com/apache/arrow-rs/issues/5454
- https://github.com/apache/arrow-rs/pull/5455